src/main/scala/is/hail/io/vcf/ExportVCF.scala

```diff
@@ -38,6 +38,23 @@ object ExportVCF {
     }
   }
 
+  def emitGenotype(f: Field, sb: StringBuilder, value: Annotation) {
+    if (value == null)
+      sb += '.'
+    else
+      f.typ match {
+        case it: TIterable =>
+          val arr = value.asInstanceOf[Iterable[_]]
+          if (arr.isEmpty) {
+            sb += '.'
```

Comment:
The VCF spec doesn't say anything. Missing value is definitely ".". The Htsjdk code to parse the PL field splits on ",". So an empty string will return an empty list. I'll change my code to output nothing if it's an empty array.